assets/javascripts/discourse/controllers/admin-wizards-wizard-show.js.es6

```diff
@@ -36,6 +36,10 @@ export default Controller.extend({
 
   @discourseComputed("wizard.id")
   wizardUrl(wizardId) {
+    let baseUrl = window.location.href.split("/admin");
+    if (baseUrl.length > 1) {
```

Comment:
Won't this always be true?, e.g.

```
foo.com/admin/wizards/wizard/test = [ 'foo.com', 'wizards/wizard/test' ]
bar.foo.com/admin/wizards/wizard/test = [ 'bar.foo.com', 'wizards/wizard/test' ]
bar.foo.com/path/admin/wizards/wizard/test = [ 'bar.foo.com/path', 'wizards/wizard/test' ]
```

Reply:
Thanks for looking into this @angusmcleod !

In cases where the page is displayed in the `admin` page view, yes this will always be true. 

I put a catch guard just in case the page is not displayed in the `admin` path. I'm still discovering the behaviour of Discourse plugin JS files, and I guess the `admin` in the file name means that it will only be displayed in the admin view?.  I have updated the code to remove the catch/guard.  
Thanks

Reply:
Great resources, thank @angusmcleod